tasks/main.yml

```diff
@@ -42,12 +42,18 @@
     - cartridge-replicasets
     - cartridge-config
 
+- name: Calculate hosts list
```

Comment:
Please, don't name machines `hosts`. Ansible `hosts` are instances in this role, I think we should use "machine" word to underline that its physical servers, not instances.

Reply:
Please, check new variant of variables names